internal/cli/atlas/deployments/list.go

```diff
@@ -144,20 +85,14 @@ func ListBuilder() *cobra.Command {
 			"output": listTemplate,
 		},
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			opts.config = config.Default()
-			opts.CredStore = config.Default()
-			log.SetWriter(cmd.OutOrStdout())
-
 			if err := opts.PreRunE(
-				opts.initStore(cmd.Context()),
-				func() error { return opts.defaultSetter.InitStore(cmd.Context()) },
-				opts.InitOutput(log.Writer(), listTemplate)); err != nil {
+				func() error { return opts.DefaultSetter.InitStore(cmd.Context()) },
```

Comment:
do you still need to do it? Isn't it part of `opts.InitStore()`?

Reply:
yeah not needed, was planning to clean up command by command in next PR, but just removed